README.md

```diff
@@ -1,2 +1,54 @@
-# Portfolio-setup-mobile-microverse
-For the first milestone in the process of creating my portfolio website, I will:  set up a new repository and prepare it for development using best practices (e.g. linters). build the first 2 sections of the mobile website using the template I chose in the previous step.
+
+# Portfolio: setup and mobile version skeleton
+
+>For the first milestone in the process of creating my portfolio website, I will:
+set up a new repository and prepare it for development using best practices (e.g. linters).
+build the first 2 sections of the mobile website using the template I chose in the previous step.
+
+
+## Built With
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](http://127.0.0.1:5502/index.html)
+
+
+## Getting Started
+
+To Create A Portfolio from this Repository feel free to contact me.
+
+To get a local copy up and running follow these simple steps.
+
+you can clone this repo by typing git clone git@github.com:peterboro/Portfolio-setup-and-mobile-microverse.git.
+type cd Portfolio-setup-and-mobile-microverse to access the project on terminal.
+
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@peterboro]
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+- GitHub: [@peterboro]
```

Comment:
- [ ] I think here you didn't update the MIT file as the previous code reviewer asked so please add it to your project you can see the template here and customize it to your information [MIT](https://github.com/Reem-lab/MyPortfolio/blob/main/LICENSE).👌🏻